docs/0004-make-authentication-configurable-at-server-startup.md

```diff
@@ -0,0 +1,247 @@
+# 4. Make Authentication Configurable at Server Startup
+
+Date: 2020-06-29
+
+## Status
+
+Accepted
```

Comment:
everything in this comment belongs here https://github.com/azavea/franklin/pull/313/files#r450500909